doc/ref_arch/openstack/README.md

```diff
@@ -18,6 +18,7 @@ This is Openstack based Reference Architecture (RA-1)
 | 1.0 | 20th September 2019 | Botrange Release|
 | 2.0-alpha | 10th January 2020 | Snezka Release|
 | 2.0 | 15th May 2020 | Baldy Release|
+| 3.0| 25th Sep 2020 | Baraque Release|
```

Comment:
ditto

Reply:
fixed in https://github.com/cntt-n/CNTT/pull/1739/commits/02d57baad26960976fd5e17ff9949b5a8d119cd7